requirements.txt

```diff
@@ -1,5 +1,5 @@
 openwisp-controller~=0.7.post1
-influxdb>=5.2,<5.3
+influxdb>=5.3,<5.4
```

Comment:
let's switch to the form `~=5.3`, which is better when using a tight range like this.

If you could also convert the other requirements but in a separate (and possibly previous) commit it would be great!

Reply:
> If you could also convert the other requirements but in a separate (and possibly previous) commit it would be great!

Created #228 and there is a query there.